compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -769,6 +769,12 @@ error.resource.function.invalid.return.type=\
 error.remote.in.non.object.function=\
   remote modifier not allowed in non-object attached function {0}
 
+error.unsupported.resource.function.declaration.in.object.type=\
+  unsupported resource function declaration in object type definition
```

Shall we use method instead?

```suggestion
  resource method declarations are not allowed in an object type definition
```